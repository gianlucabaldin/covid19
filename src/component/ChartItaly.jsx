import React from 'react';
import { Box } from '@material-ui/core';

const ChartItaly = ({ data }) => {
  return (
    <Box
      display="flex"
      justifyContent="center"
      m={1}
      p={1}
      bgcolor="background.paper"
    >
      {data.confirmed}
      <br />
      {data.recovered}
      <br />
      {data.deaths}
      wgnfoeurb
    </Box>
  );
};

ChartItaly.defaultProps = {
  data: {
    confirmed: 0,
    recovered: 0,
    deaths: 0,
  },
};

export default ChartItaly;
