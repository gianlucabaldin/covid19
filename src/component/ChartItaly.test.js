import React from 'react';
import { shallow, mount, render } from 'enzyme';
import ChartItaly from './ChartItaly';

describe('ChartItaly', () => {
  let wrapper = shallow(<ChartItaly />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('renders with default props', () => {
    wrapper = mount(<ChartItaly />);
    expect(wrapper.find('Styled(MuiBox)').length).toEqual(1);
  });
});
