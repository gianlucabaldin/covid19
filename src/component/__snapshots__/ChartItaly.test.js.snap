// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartItaly renders properly 1`] = `
<Styled(MuiBox)
  bgcolor="background.paper"
  display="flex"
  justifyContent="center"
  m={1}
  p={1}
>
  0
  <br />
  0
  <br />
  0
  wgnfoeurb
</Styled(MuiBox)>
`;
